test/unit/test_boundary_conditions.py

```diff
@@ -507,6 +507,13 @@ def test_recomb_flux():
     my_BC = festim.RecombinationFlux(surfaces=[0], Kr_0=expr, E_Kr=expr, order=2)
     my_BC.create_form(T, c)
 
+def test_mass_flux():
```

Comment:
```suggestion

def test_mass_flux():
```

black requires 2 lines between functions